src/components/file-uploader/file-uploader-shell.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2019, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, property, customElement, LitElement } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import styles from './file-uploader.scss';
+
+const { prefix } = settings;
+
+/**
+ * The shell UI for file uploader.
+ * @element bx-file-uploader-shell
+ * @slot helper-text The helper text.
+ * @slot label-text The label text.
+ */
+@customElement(`${prefix}-file-uploader-shell`)
```

Comment:
I think we should be good to just have
```suggestion
@customElement(`${prefix}-file-uploader`)
```
for this - the rest of the components just slot into this one as needed anyway.

Reply:
@cal-smith Thank you for your thoughts - Makes sense to me, updated.